.circleci/config.yml

```diff
@@ -43,7 +43,10 @@ jobs:
     <<: *job_executor
     steps:
       - checkout
-      - node/install-packages:
+      - node/install-packages: # https://circleci.com/developer/orbs/orb/circleci/node#commands-install-packages
+          cache-only-lockfile: true
+          check-cache: always # yarn berry zero installs
+          cache-version: v2
```

Comment:
@lordpixel adding this fixed it. This is something new in the node orb, which I also updated. If we ever run into the scenario again where we see a cache that is failing to expand due to permissions, we can change this value in the cache version and it will cause the install to not find any previous cached installs and run from scratch.